tests/acceptance/administrator/AdministratorRedirectCest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_redirect
+ *
+ * @copyright   Copyright (C) 2005 - 2015 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+class AdministratorRedirectCest
+{
+	public function __construct()
+	{
+		$this->faker = Faker\Factory::create();
+		//$this->title  = 'Redirect' . $this->faker->randomNumber();
+		//$this->oldurl  = $this->faker->url();
+		$this->oldurl  = 'test'.$this->faker->randomNumber().'.htm';
+		$this->newurl = 'http://www.test.com';
+	}
+
+	public function administratorVerifyAvailableTab(AcceptanceTester $I)
+	{
+		$I->am('Administrator');
+		$I->wantToTest('Redirect Edit View Tab');
+
+		$I->doAdministratorLogin();
+
+		$I->amGoingTo('Navigate to Redirect page in /administrator/ and verify the Tab');
+		$I->amOnPage('administrator/index.php?option=com_redirect');
+		$I->waitForText('Redirects: Links', '30', ['css' => 'h1']);
```

Comment:
I would like to give two suggestions here,
1. Instead of css , the class locater will perform faster. You should try: ['class' => 'page-title']
2. I think instead of '30' it should be 30 as it is timeout and should be integer.